JamGame/Assets/Scripts/Employee/StressMeter/Inspector.cs

```diff
@@ -0,0 +1,140 @@
+﻿#if UNITY_EDITOR
+using Employee;
+using Employee.StressMeter;
+using UnityEditor;
+using UnityEngine;
+
+namespace Assets.Scripts.Employee.StressMeter
+{
+    [CustomEditor(typeof(StressMeterImpl))]
+    [CanEditMultipleObjects]
+    public class Inspector : Editor
```

Comment:
Custom editor don't work with Odin Inspector, so make with Odin, partial class with bunch of attributes or create custom [value drawer](https://odininspector.com/tutorials/how-to-create-custom-drawers-using-odin/how-to-make-a-healthbar-attribute) for example. Also many of things you can do with groups attributes for layouting.

Reply:
Also, when `StressMeterImpl` re-drawed in editor, prefab always saves himself.

Reply:
Added check to not make object dirty each frame

Reply:
Discussed